docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeLogMeanVariance.cs

```diff
@@ -12,15 +12,15 @@ public class NormalizeLogMeanVariance
     {
         public static void Example()
         {
-            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging,
             // as well as the source of randomness.
             var mlContext = new MLContext();
             var samples = new List<DataPoint>()
             {
-                new DataPoint(){ Features = new float[4] { 1, 1, 3, 0} },
-                new DataPoint(){ Features = new float[4] { 2, 2, 2, 0} },
-                new DataPoint(){ Features = new float[4] { 0, 0, 1, 0} },
-                new DataPoint(){ Features = new float[4] {-1,-1,-1, 1} }
+                new DataPoint(){ Features = new float[5] { 1, 1, 3, 0, float.MaxValue } },
```

Comment:
Is this change related to `FixZero`?

Reply:
It's related to the original issue posted by Lisa. They were encountering an issue with the MeanVariance normalizer which does not handle data that looks like `float.MaxValue`, `float.MinValue`. I think it would be useful to show that LogMeanVariance normalizer accepts this data in the related sample.

---
In reply to: [298254220](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298254220) [](ancestors = 298254220)